tests/integration/data_quality_test.py

```diff
@@ -33,21 +30,16 @@ def data_dir(request) -> Path:
 )
 def extracted(metadata_dir, data_dir, request) -> ExtractOutput:
     form, year = request.param
-    tables = get_fact_tables(
+    return extract(
         taxonomy_path=TAXONOMY_MAP[form],
         form_number=form,
         db_path="path",
-        metadata_path=Path(metadata_dir) / "metadata.json",
-    )
-    instance_builders = get_instances(data_dir / f"ferc{form}-xbrl-{year}.zip")
-    instances = [ib.parse() for ib in instance_builders]
-    filings, stats = extract(
-        instances=instances,
-        tables=tables,
-        batch_size=max(1, len(instances) // 5),
-    )
-    return ExtractOutput(
-        tables=tables, instances=instances, filings=filings, stats=stats
+        archive_path=None,
+        metadata_path=metadata_dir / "metadata.json",
+        datapackage_path=metadata_dir / "datapackage.json",
+        instance_path=data_dir / f"ferc{form}-xbrl-{year}.zip",
+        workers=None,
+        batch_size=2,
```

Comment:
TODO: maybe we refactor this to `num-batches`?

Reply:
I guess `batch_size` is better for CLI (you don't know how many files there are) but `num_batches` is more helpful for test (you just care that it gets split up in some way)... I'll eat the slightly more annoying testing.